File: Scripts/src/icarus/model/EL/table/TGROUP/THEAD.js
/** @module */
import TGROUP, { MODEL, TD, TH, TR } from './TGROUP.js';
/** A TABLE Header Group */
export default class THEAD extends TGROUP {
    /** TBODY Constructor
        @param {CONTAINER} node Node
        @param {ContainerModel} model Model
    */
    constructor(node, model) {
        super(node, 'THEAD', model);		
        this.addClass('table-header');
    }
}
export { MODEL, TD, TH, TR }

File: Models/Icarus/Elements/THEAD.cs
using ICARUS.Models.Icarus.Elements.Attributes;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace ICARUS.Models.Icarus.Elements {

    /// <summary>
    /// Represents a table header
    /// </summary>
    public class THEAD : Container {

        /// <summary>
        /// An article is the basic building block for nearly all pages.  
        /// Articles are made up of a collection of SECTIONS
        /// </summary>
        /// <param name="attributes"></param>
        public THEAD() : base("THEAD", new MODEL() {
            label = "THEAD"
        }) {

        }

        /// <summary>
        /// Construct a THEAD based on the provided Form Post
        /// </summary>
        /// <param name="formPost"></param>
        public THEAD(FormPost formPost) : base("THEAD", formPost) {

        }
    }
}